lib/matplotlib/table.py

```diff
@@ -146,6 +147,57 @@ def set_text_props(self, **kwargs):
         self._text.update(kwargs)
 
 
+class CustomCell(Cell):
+    """
+    A subclass of Cell where the sides may be visibly toggled.
+
+    """
+
+    EDGES = 'BRTL'
+    EDGE_ALIASES = {'open':         '',
```

Comment:
I think that a leading underscore would be fine.  They really are implementation details.